donate/utils.py

```diff
@@ -0,0 +1,36 @@
+from django.core.mail import send_mail, BadHeaderError
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+import smtplib
+
+
+def send_transaction_email(user_email, transaction_details):
+    # Render the email content using a template
+    subject = 'Spoken Tutorial - School Donation Transaction Deatils'
```

Comment:
```suggestion
    subject = 'Spoken Tutorial - School Donation Transaction Details'
```
Fix the spelling of Details